GitUI/CommandsDialogs/SettingsDialog/Pages/GitExtensionsSettingsPage.Designer.cs

```diff
@@ -329,6 +331,17 @@ private void InitializeComponent()
             this.chkShowGitStatusInToolbar.Text = "Show number of changed files on commit button (restart required)";
             this.chkShowGitStatusInToolbar.UseVisualStyleBackColor = true;
             // 
+            // chkShowGitStatusForArtificialCommits
+            // 
+            this.chkShowGitStatusForArtificialCommits.AutoSize = true;
+            this.tableLayoutPanelPerformance.SetColumnSpan(this.chkShowGitStatusForArtificialCommits, 2);
+            this.chkShowGitStatusForArtificialCommits.Location = new System.Drawing.Point(3, 3);
+            this.chkShowGitStatusForArtificialCommits.Name = "chkShowGitStatusForArtificialCommits";
+            this.chkShowGitStatusForArtificialCommits.Size = new System.Drawing.Size(451, 17);
+            this.chkShowGitStatusForArtificialCommits.TabIndex = 1;
```

Comment:
Could you please make sure the tab order on the page is in order

Reply:
Updated. I liked the italics myself....

Reply:
The revision grid is very busy with different colors and fonts. 
All of that may/causes a visual overload. I prefer not to increasing the visual entropy, unless necessary.